### ISSUE TITLE
Trouble getting this to run on Windows

### ISSUE THREAD

**Issue by meetloaf13** [BukkitPE/Installation#1]
Hey guys,

Really excited to try this out.  I've been running ClearSky for several weeks and learned about your version of PM.  

Interested in trying out the nether on my server =]

I'm on Windows, I simply copied my working clearsky server directory, deleted the bin folder, deleted the pocketmine.cmd, deleted the pocketmine-mp.phar, pasted in your /bin, pasted in your bukketPE.phar & start.cmd.

When ever I run start.cmd whether by double clicking, right clicking run as admin, or directly from the console, the PocketMine window starts and stops immediately.

If I go over to my copied directory, and launch PocketMine there, the server starts up fine.

Am I doing something wrong?  

I even tried making a directory that contains only the following:

-/bin
-BukkitPE.phar
-Start.cmd

Any ideas how to get me up and running?  

Thanks!

**Comment by AndreTheGamer** [comment]
Hi, download the zip file of BukkitPE from the release page. Or you can click that link [Here](https://github.com/BukkitPE/BukkitPE/releases/download/1.0.0/BukkitPE.zip) Then extract BukkitPE and copy the bin foldre to the installation and run start.cmd and it should work :) Keep me updated if you run into problems.

**Comment by meetloaf13** [comment]
Yeah, I've tried starting from scratch just now.  No dice, same behavior =\
http://i.imgur.com/NsM5EYv.png

**Comment by meetloaf13** [comment]
I can't seem to find an error log or anything else to send your way on this one =\

Running Windows 10 x64 with working ClearSky, if that helps

**Comment by AndreTheGamer** [comment]
Ok, download a whole new copy, then use you're existing bin folder that you are using for ClearSky, just copy and paste it in BukkitPE folder. After run start.cmd it should work now.

**Comment by meetloaf13** [comment]
Man, same problem =\  

Really wish I had a log file or something, but it pretty much starts/stops instantaneously.

**Comment by AndreTheGamer** [comment]
Add me on skype, andreleon911 So i can better assit you.

**Comment by meetloaf13** [comment]
done, thanks

**Comment by meetloaf13** [comment]
For anyone following, we fixed it by simply renaming the phar to 'PocketMine-MP.phar'

Thanks Andre, super helpful.

**Comment by AndreTheGamer** [comment]
Sure, no problem man. :)
